acceptance-tests/tests/simple-permissioning-smart-contract/README.md

```diff
@@ -123,3 +123,15 @@ Compiling ./contracts/SimplePermissioning.sol...
 
   10 passing (1s)
 ```
+
+##Permissioning Contract Updating Steps
+Should these contracts require updating, please ensure that the compiled bytecode in the Permissioning acceptance tests genesis file has been updated with the new bytecode.
+
+One simple update procedure is as follows:
+1. Load existing contract in [Remix IDE](https://remix.ethereum.org/).
+2. Enable the _"SOLIDITY COMPILER"_ plugin.
+3. Make required changes.
```

Comment:
should we say to turn optimisation on since it makes the bytecode quite different?

Reply:
Actually, we should turn off optimisation because it's for production contracts any may cause debugging issues: https://remix.readthedocs.io/en/stable/settings_tab.html